### PR TITLE
Pull Request to add ZergNet to amp-embed

### PR DESCRIPTION
**Pull request by zergnetjake**

### CODE REVIEW

File: ads/zergnet.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (erwinmombay) [inline ads/zergnet.md]: 2016

File: ads/zergnet.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import {writeScript, checkData, validateDataExists} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function zergnet(global, data) {
+    checkData(data, ['zergid']);
+    validateDataExists(data, ['zergid']);
+    global.zergnetWidgetId = data.zergid;
```
Comment (erwinmombay) [inline ads/zergnet.js]: you'll need to add these two properties here https://github.com/ampproject/amphtml/blob/master/ads/ads.extern.js

Comment (zergnetjake) [inline ads/zergnet.js]: Just add data.zergid and var zergnetWidgetId at the end?

Comment (erwinmombay) [inline ads/zergnet.js]: yes, i think that should be fine.

`data.zergid`
`window.zergnetWidgetId`

/cc @cramforce

**Comment by jridgewell** [comment]
Re: https://github.com/ampproject/amphtml/pull/4009

**Comment by zergnetjake** [comment]
I signed it!  I made the original request as ZergNet, the COO signed it as 7Hops.com, Inc.

**Comment by zergnetjake** [comment]
Apparently my git was configured with my personal address on this repo, what should I do to get the cla/google check to complete?  The CLA has been signed by my work email.

**Comment by zergnetjake** [comment]
I can delete this fork and resubmit if necessary

**Comment by erwinmombay** [comment]
i think we could still make that work but i will defer to @cramforce

**Pull request closed without merging**